source/extensions/filters/http/grpc_http1_reverse_bridge/filter.cc

```diff
@@ -37,8 +38,20 @@ Grpc::Status::GrpcStatus grpcStatusFromHeaders(Http::HeaderMap& headers) {
   }
 }
 
+std::string badContentTypeMessage(const Http::HeaderMap& headers) {
+  if (headers.ContentType() != nullptr) {
+    return fmt::format(
+        "envoy reverse bridge: upstream responded with unsupported content-type {}, status code {}",
+        headers.ContentType()->value().getStringView(), headers.Status()->value().getStringView());
+  } else {
+    return fmt::format(
+        "envoy reverse bridge: upstream responded with no content-type header, status code {}",
+        headers.Status()->value().getStringView());
+  }
+}
+
 void adjustContentLength(Http::HeaderMap& headers,
-                         std::function<uint64_t(uint64_t value)> adjustment) {
+                         const std::function<uint64_t(uint64_t value)>& adjustment) {
```

Comment:
this was an unrelated change that my IDE linter suggested, can revert if  we want to isolate the  change